subprojects/instant-execution/src/integTest/groovy/org/gradle/instantexecution/inputs/undeclared/UndeclaredBuildInputsTestKitInjectedJavaPluginIntegrationTest.groovy

```diff
@@ -43,6 +46,11 @@ class UndeclaredBuildInputsTestKitInjectedJavaPluginIntegrationTest extends Abst
         return "plugin 'sneaky'"
     }
 
+    @Rule
+    SetSystemProperties setSystemProperties = new SetSystemProperties(
+        (NativeServices.NATIVE_DIR_OVERRIDE): buildContext.nativeServicesDir.absolutePath
+    )
```

Comment:
Should this change be in this PR?

Reply:
Yes, I ran into some problems. See the discussion here: https://gradle.slack.com/archives/C03A0MT6S/p1592825171344300

I could move this to a separate PR if you think it is necessary.

Reply:
I see now how the problems are related. I think it's okay to leave this here.